samples/config/prod/deploy.sh

```diff
@@ -99,8 +99,8 @@ deploy_charts()
             ;;
         esac
 
-        helm install --name ${c}-${NAMESPACE} -f type/common.yaml -f type/${TYPE}/${c}.yaml \
-            --namespace=${NAMESPACE} ../../helm/${chart}
+        helm install --name ${c}-${NAMESPACE} -f common.yaml -f ${TYPE}/${c}.yaml \
+            --namespace=${NAMESPACE} ../../../helm/${chart}
```

Comment:
We should test for the type specific yaml, and if it does not exist (not all will need it) - dont reference it.